modules/apps/client-extension/client-extension-web/src/main/java/com/liferay/client/extension/web/internal/display/context/util/CETLabelUtil.java

```diff
@@ -32,6 +32,12 @@ public static String getAddLabel(
 		return _getLabel(httpServletRequest, "add-", type);
 	}
 
+	public static String getHelpNameLabel(
```

Comment:
This should probably be `getHelpLabel` (as `getNewLabel`, for example)

Reply:
I was following the example of `getTypeLabel` – is it correct to have the same name for the "function" and the "callback" ?

<img width="690" alt="image" src="https://user-images.githubusercontent.com/46778114/186353498-305abdc6-a788-4b3b-b59f-658ec71f0493.png">